WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.kt

```diff
@@ -0,0 +1,293 @@
+package org.wordpress.android.ui.notifications
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.os.Parcelable
+import android.text.Html
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.google.android.material.appbar.AppBarLayout.LayoutParams
+import com.google.android.material.tabs.TabLayout.OnTabSelectedListener
+import com.google.android.material.tabs.TabLayout.Tab
+import kotlinx.android.synthetic.main.notifications_list_fragment.*
+import org.greenrobot.eventbus.EventBus
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.analytics.AnalyticsTracker
+import org.wordpress.android.analytics.AnalyticsTracker.NOTIFICATIONS_SELECTED_FILTER
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.NOTIFICATION_TAPPED_SEGMENTED_CONTROL
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.AccountStore
+import org.wordpress.android.ui.ActivityLauncher
+import org.wordpress.android.ui.JetpackConnectionSource.NOTIFICATIONS
+import org.wordpress.android.ui.JetpackConnectionWebViewActivity
+import org.wordpress.android.ui.RequestCodes
+import org.wordpress.android.ui.WPWebViewActivity
+import org.wordpress.android.ui.main.WPMainActivity
+import org.wordpress.android.ui.notifications.NotificationEvents.NotificationsUnseenStatus
+import org.wordpress.android.ui.notifications.adapters.NotesAdapter.FILTERS
+import org.wordpress.android.ui.notifications.adapters.NotesAdapter.FILTERS.FILTER_ALL
+import org.wordpress.android.ui.notifications.adapters.NotesAdapter.FILTERS.FILTER_COMMENT
+import org.wordpress.android.ui.notifications.adapters.NotesAdapter.FILTERS.FILTER_FOLLOW
+import org.wordpress.android.ui.notifications.adapters.NotesAdapter.FILTERS.FILTER_LIKE
+import org.wordpress.android.ui.notifications.adapters.NotesAdapter.FILTERS.FILTER_UNREAD
+import org.wordpress.android.ui.notifications.services.NotificationsUpdateServiceStarter
+import org.wordpress.android.ui.notifications.services.NotificationsUpdateServiceStarter.IS_TAPPED_ON_NOTIFICATION
+import org.wordpress.android.ui.stats.StatsConnectJetpackActivity
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T.NOTIFS
+import org.wordpress.android.util.NetworkUtils
+import org.wordpress.android.util.WPUrlUtils
+import java.util.HashMap
+import javax.inject.Inject
+
+class NotificationsListFragment : Fragment() {
+    private var mShouldRefreshNotifications = false
+    private var mLastTabPosition = 0
```

Comment:
I'd remove the Hungarian notation in the Kotlin classes

Reply:
Fixed in e116fff29ed17a8a1eb8db9e16bbe64e06246e55